node/bft/src/helpers/channels.rs

```diff
@@ -60,6 +60,7 @@ pub fn init_consensus_channels<N: Network>() -> (ConsensusSender<N>, ConsensusRe
     (sender, receiver)
 }
 
+/// "Interface" that enables, for example, sending data from storage to the BFT logic.
```

Comment:
Not sure where this came from, but we can probably include it in this PR.

Reply:
okey, thank you

Reply:
Perhaps this is more precise? 

```suggestion
/// Outbound channels the primary uses to communicate with the BFT.
```

Reply:
@niklaslong ready!